scripts/ci-build-azure-ccm.sh

```diff
@@ -69,16 +68,16 @@ main() {
         echo "Building and pushing Linux and Windows amd64 Azure ACR credential provider"
         make -C "${AZURE_CLOUD_PROVIDER_ROOT}" bin/azure-acr-credential-provider bin/azure-acr-credential-provider.exe
 
-        if [[ "$(az storage container exists --name "${AZURE_BLOB_CONTAINER_NAME}" --query exists --output tsv)" == "false" ]]; then
+        if [[ "$(az storage container exists ${ENABLE_AUTH_MODE_LOGIN:+"--auth-mode login"} --name "${AZURE_BLOB_CONTAINER_NAME}" --query exists --output tsv)" == "false" ]]; then
```

Comment:
`${ENABLE_AUTH_MODE_LOGIN:+"--auth-mode login"}` is the penultimate workaround in the list of exceptions. 
[SC2086#exceptions](https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions)

Note that the value of `ENABLE_AUTH_MODE_LOGIN` immaterial as long as it is set for `${ENABLE_AUTH_MODE_LOGIN:+"--auth-mode login"}` evaluate to `"--auth-mode login"`

Reply:
TIL